<ui version="4.0" >
 <class>DiskClass</class>
 <widget class="QWidget" name="DiskClass" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>595</width>
    <height>448</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Disk</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2" >
   <item>
    <widget class="QGroupBox" name="groupBox" >
     <property name="font" >
      <font>
       <family>Book Antiqua</family>
       <pointsize>20</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
       <underline>false</underline>
       <strikeout>false</strikeout>
      </font>
     </property>
     <property name="title" >
      <string>Format Removeable HDD</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout" >
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4" >
        <item>
         <widget class="QProgressBar" name="progressBar" >
          <property name="value" >
           <number>0</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_format" >
          <property name="maximumSize" >
           <size>
            <width>120</width>
            <height>35</height>
           </size>
          </property>
          <property name="font" >
           <font>
            <family>Book Antiqua</family>
            <pointsize>18</pointsize>
            <weight>50</weight>
            <italic>false</italic>
            <bold>false</bold>
            <underline>false</underline>
            <strikeout>false</strikeout>
           </font>
          </property>
          <property name="text" >
           <string>Format</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout" >
        <item>
         <widget class="QTextBrowser" name="textBrowser" />
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="label" >
        <property name="alignment" >
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <resources/>
 <connections/>
</ui>
